tests/unit/amo/components/TestConfirmationDialog.js

```diff
@@ -42,77 +26,65 @@ describe(__filename, () => {
     expect(cancelButton).toHaveClassName('ConfirmationDialog-cancel-button');
     expect(cancelButton).toHaveProp('buttonType', 'cancel');
     expect(cancelButton.children()).toHaveText('Cancel');
+    */
   });
 
   it('renders a custom className', () => {
     const className = 'MyComponent';
-    const root = render({ className });
-    expect(root).toHaveClassName(className);
-    expect(root).toHaveClassName('ConfirmationDialog');
+    render({ className });
+    expect(screen.getByClassName(className)).toHaveClass('ConfirmationDialog');
   });
 
-  it.each([true, false])(
-    'can configure buttons as puffy=%s',
-    (puffyButtons) => {
-      const root = render({ puffyButtons });
-
-      expect(root.find('.ConfirmationDialog-cancel-button')).toHaveProp(
-        'puffy',
-        puffyButtons,
-      );
-      expect(root.find('.ConfirmationDialog-confirm-button')).toHaveProp(
-        'puffy',
-        puffyButtons,
-      );
-    },
-  );
+  it('can configure buttons as non puffy', () => {
+    render({ puffyButtons: false });
 
-  it('calls onConfirm() when user clicks Confirm', () => {
-    const onConfirm = sinon.spy();
-    const root = render({ onConfirm });
+    expect(
+      screen.getByClassName('ConfirmationDialog-cancel-button'),
+    ).not.toHaveClass('Button--puffy');
+    expect(
+      screen.getByClassName('ConfirmationDialog-confirm-button'),
+    ).not.toHaveClass('Button--puffy');
+  });
 
-    const event = createFakeEvent();
-    root.find('.ConfirmationDialog-confirm-button').simulate('click', event);
+  it('can configure buttons as puffy', () => {
```

Comment:
Nit: I would move this test above the negative test. This allows someone reading the file to see that puffy buttons do in fact have the `Button--puffy` class, and then it becomes obvious that the negative test is correct in that it's testing for the absence of the same class. Negative tests are tricky because there can be lots of reasons why a class might be absent, so the test setup should be identical for the positive and negative tests (except for the one thing that is changing, of course) to make it easier for a reviewer to be confident that the negative test is correct.

This is an advantage to using the `it.each` format, but I agree in this case it makes more sense to break it up into two tests.